api/app.js

```diff
@@ -23,10 +23,10 @@ db.on('error', function(err){
 
 
 var indexRouter = require('./routes/index');
-var usersRouter = require('./routes/users');
+var usersRouter = require('./routes/usersController');
 var picsRouter = require('./routes/picsController');
 var loginRouter = require('./routes/login');
-var users = require('./routes/users');
+var users = require('./routes/usersController');
 
```

Comment:
why two routers

Reply:
the one in red means it will get deleted so I will only have on router

Reply:
But you have two: one users and one usersRouter

Reply:
Oh I see, I will delete one.